### PR TITLE
Update Release Leads for 0.20

### PR DESCRIPTION
**Pull request by julz**
Following the [Release Leads process](https://github.com/knative/pkg/blob/master/RELEASE-LEADS.md): the release is now cut, passing on the torch to @nak3 and @slinkydeveloper!.

/assign @n3wscott @mattmoor

### CODE REVIEW

**Comment by nak3** [comment]
LGTM :smile:

**Comment by slinkydeveloper** [comment]
0.20 will be a rocky one!!! :laughing:  /lgtm

**Comment by mattmoor** [comment]
This LGTM, are we ready to pull the trigger on it?  cc @n3wscott

File: peribolos/knative-sandbox.yaml [hunk 1]
```diff
@@ -565,8 +565,8 @@ orgs:
           Knative Release Leads:
             description: Active member of the Knative Release Leads rotation.
             maintainers:
-            - julz
-            - n3wscott
```
Comment (n3wscott) [inline peribolos/knative-sandbox.yaml:569]: good night, sweet access

Comment (julz) [reply peribolos/knative-sandbox.yaml:569]: you mean we don't get to keep it till January 20th? 😢

**Pull request merged**